<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertiesPane</class>
 <widget class="QMainWindow" name="PropertiesPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>383</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Window Properties</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actnLocate"/>
   <addaction name="actnFlash"/>
   <addaction name="separator"/>
   <addaction name="actnSave"/>
  </widget>
  <action name="actnLocate">
   <property name="icon">
    <iconset resource="../../qt_resources.qrc">
     <normaloff>:/img/locate.png</normaloff>:/img/locate.png</iconset>
   </property>
   <property name="text">
    <string>Locate</string>
   </property>
   <property name="toolTip">
    <string>Shows the window in the hierarchy tree</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actnSave">
   <property name="icon">
    <iconset resource="../../qt_resources.qrc">
     <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Window Properties</string>
   </property>
   <property name="toolTip">
    <string>Save the window properties to a file in either text or xml format</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actnFlash">
   <property name="icon">
    <iconset resource="../../qt_resources.qrc">
     <normaloff>:/img/flash.png</normaloff>:/img/flash.png</iconset>
   </property>
   <property name="text">
    <string>Flash</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../qt_resources.qrc"/>
 </resources>
 <connections/>
</ui>
